app/views/recipe_foods/general_shopping.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= render partial: 'layouts/section_title', locals: { section_title: 'General Shopping List' } %>
```

Comment:
- [ ] [OPTIONAL] General shopping list view is throwing an error. I suggest you remove it as this feature is required by a group of 3 contributors.

**Page error:**
![image](https://user-images.githubusercontent.com/12745474/226096538-5b4f7784-8cf1-4ed8-a42d-795b7f1b399e.png)

Reply:
Option removed as required.